Formatter/Pool.php

```diff
@@ -34,15 +34,13 @@ class Pool
     protected $logger;
 
     /**
-     * @param LoggerInterface|null $logger
      * @param string|null          $defaultFormatter
+     * @param LoggerInterface|null $logger
      */
-    public function __construct(LoggerInterface $logger = null, $defaultFormatter = null)
+    public function __construct($defaultFormatter, LoggerInterface $logger = null)
```

Comment:
> Actually I am not sure why I didn't add any

Probably because this was quite old ;)

I'll remove the null in the signature then